From 42188b7698da83a2e43622d375c6c6f1cedf4c5c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Fri, 13 Sep 2019 05:17:03 +0200
Subject: [PATCH 135/152] folderIcon: Remove duplicated addMenu call

The RenameFolderMenu is added already to its menu manager, so no need to
repeat the operation.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/720
---
 js/ui/appDisplay.js | 2 --
 1 file changed, 2 deletions(-)

diff --git a/js/ui/appDisplay.js b/js/ui/appDisplay.js
index 8571d86b9..21469d1f9 100644
--- a/js/ui/appDisplay.js
+++ b/js/ui/appDisplay.js
@@ -1696,8 +1696,6 @@ var FolderIcon = class FolderIcon {
             this.actor.connect('destroy', () => {
                 Main.overview.disconnect(id);
             });
-
-            this._menuManager.addMenu(this._menu);
         }
 
         this.actor.set_hover(true);
-- 
2.23.0

